coreos-base/coreos/coreos-0.0.1.ebuild

```diff
@@ -153,7 +153,7 @@ RDEPEND="${RDEPEND}
 	sys-fs/e2fsprogs
 	sys-fs/mdadm
 	sys-fs/xfsprogs
-	sys-kernel/coreos-firmware
+	amd64? ( sys-kernel/coreos-firmware )
```

Comment:
The 4.x kernel config does happen to have a driver that uses firmware (although admittedly the driver was enabled just to make the ebuild work). Perhaps a `firmware` use flag would be appropriate?